src/status_im/ui/screens/chat/views.cljs

```diff
@@ -109,9 +110,48 @@
 
 (defn empty-chat-container
   []
-  [react/view style/empty-chat-container
-   [react/text {:style style/empty-chat-text}
-    (i18n/label :t/empty-chat-description)]])
+  (let [{:keys [color chat-id chat-name]}
+        @(re-frame/subscribe [:chats/current-chat])
```

Comment:
if possible, try to pass parameters to the function rather than subscribe within

Reply:
I see
gonna subscribe it in letsubs in message-view component and pass it to empty-chat-container

Reply:
done!